Extensions/Fetch/FetchService.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace FroshTemplateExtensions\Extensions\Fetch;
+
+use Doctrine\DBAL\Connection;
+
+class FetchService implements FetchServiceInterface
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * @var array
+     */
+    private $_cache = [];
```

Comment:
Why the property begins with underscore? :sweat_smile:

Reply:
No actual reason 😂 Derives from an outdated naming convention for private properties. I still like to use it for private properties that provide no functional use and contain pure data.